_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -0,0 +1,248 @@
+
+```html
+<div role="table">
+	<div>
+		<div>
+			<div role="row">
+				<div role="cell">my cell</div>
+				<div role="cell">and another cell</div>
+			</div>
+			<div role="row">
+				<div role="cell">my cell</div>
+				<div role="cell">and another cell</div>
+			</div>
+		</div>
+	</div>
+</div>
+```
+
+#### Passed example 6
+
+Element with role `grid` owns multiple instances of required owned elements.
+
+```html
+<div role="grid">
+	<div role="rowgroup">
+		<div role="row">
+			<div role="gridcell">1</div>
+			<div role="gridcell">Item 1</div>
+			<div role="gridcell">2 Nos.</div>
+		</div>
+	</div>
+	<div role="rowgroup">
+		<div role="row">
+			<div role="gridcell">1</div>
+			<div role="gridcell">Item 1</div>
+			<div role="gridcell">2 Nos.</div>
+		</div>
+	</div>
+</div>
+```
+
+#### Passed example 7
+
+Element with role `list` owns an instance of required owned element which has an implicit role of `listitem`.
+
+```html
+<div role="list">
+	<li></li>
+</div>
+```
+
+#### Passed example 8
+
+Element with role `menubar` can own multiple required owned element with role `menuitem`, `menuitemcheckbox` and `menuitmeradio`, but only one is necessary.
+
+```html
+<div role="menubar">
+	<div role="menuitem"></div>
+	<div role="menuitemcheckbox"></div>
+</div>
+```
+
+#### Passed example 9
+
+Element with role `list` contains a required `listitem`, the relationship between both elements is made possible through the `aria-owns` properties.
+
+**Note:** This test case follows the definition of [owned by](#owned-by). If implemented differently, this definition could cause differences in outcome of this test case.
+
+```html
+<div role="list" aria-owns="id1"></div>
+<div id="id1" role="listitem"></div>
+```
+
+### Failed
+
+#### Failed example 1
+
+Element with role `list` is missing required owned element `list`.
+
+```html
+<ul role="list">
+	<span> </span>
+</ul>
+```
+
+#### Failed example 2
+
+Element with role `tablist` is missing required owned element `tab`.
+
+```html
+<ol role="tablist">
+	<li role="listitem"></li>
+</ol>
+```
+
+#### Failed example 3
+
+Element with role `grid` owns required owned element `rowgroup`, which is missing its required owned element `row`.
+
+```html
+<div role="grid">
+	<div>
+		<div role="rowgroup">
+			<span>Item 1</span>
+		</div>
+		<div></div>
+	</div>
+</div>
+```
+
+#### Failed example 4
+
+The element with the semantic role of `list` owns the element with the role `tab` through the `aria-owns` property.
+
+**Note:** This test case follows the definition of [owned by](#owned-by). If implemented differently, this definition could cause differences in outcome of this test case.
+
+```html
+<div role="list" aria-owns="id2"></div>
+<div id="id2" role="tab"></div>
+```
+
+### Inapplicable
```

Comment:
One suggestion for an inapplicable example
- one element with a role that does not have required owned roles (e.g. region, progressbar)